main.tf

```diff
@@ -423,6 +432,7 @@ resource "random_string" "webhook_secret" {
 }
 
 locals {
+  service_names = compact(concat(var.service_names, [var.service_name]))
```

Comment:
this will confuse the end-users, should be only one variable, so just keep `var.service_names` and remove old one `var.service_name` and also need to update `examples` and `test` due this change

Reply:
I tried to keep it backward compatible. Let me know if you think that was a bad idea, in which case I'll remove the service_name

Reply:
remove pls `service_name`